inference-engine/ie_bridges/python/wheel/CMakeLists.txt

```diff
@@ -70,4 +70,4 @@ add_custom_command(OUTPUT ${openvino_wheel_path}
     COMMENT "Building Python wheel ${openvino_wheel_name}"
     VERBATIM)
 
-add_custom_target(ie_wheel DEPENDS ${openvino_wheel_path})
+add_custom_target(ie_wheel ALL DEPENDS ${openvino_wheel_path})
```

Comment:
Why do we need it?

Reply:
without ALL option it will never executed with ENABLE_WHEEL=ON. This adds `ie_wheel` in the default list of targets.